### DISCUSSION TITLE
🎉 Exciting Update: YOLO11 Models Now Supported in X-AnyLabeling! 🚀

### DISCUSSION THREAD

**Discussion by CVHub520** [CVHub520/X-AnyLabeling#654] [Announcements]
Dear X-AnyLabeling Community,

I'm thrilled to announce a major update to our project! 🌟 We've successfully integrated YOLO11 Det/OBB/Pose/Seg/Track models from ultralytics v8.3.0 into X-AnyLabeling.

## What's New ✨

X-AnyLabeling now supports YOLO11 models for:
- Object Detection 🔍
- Oriented Bounding Box (OBB) 📦
- Pose Estimation 🤸
- Segmentation 🧩
- Tracking 🏃‍♂️

## Why This Matters 🤔
This integration significantly expands our toolkit, allowing for more advanced and diverse labeling capabilities. Whether you're working on autonomous driving, surveillance, or any other computer vision project, these new models will enhance your labeling experience.

## How to Use 🖱️
Check out the latest release for detailed instructions on how to leverage these new YOLO11 models in your projects. Documentation has been updated to reflect these changes.

## Acknowledgements 🙏
A huge thank you to the ultralytics team for their outstanding work on YOLO11. Their continuous innovation in the field of computer vision is truly inspiring.

## Feedback and Contributions 📣
As always, your feedback is invaluable. If you encounter any issues or have suggestions for improvements, please open an issue. Pull requests for enhancements are always welcome!

Let's push the boundaries of computer vision labeling together! 💪

Happy Labeling! 

Creator of X-AnyLabeling

P.S. Spread the word! If you're excited about this update, consider giving us a star on GitHub or sharing this news with your network. (⌐■_■)

**Comment by klabklabgood** [comment]
Does X-AnyLabeling support self-trained custom YOLO11n models? I cannot find YOLO11n-OBB in the model library.

**Reply by CVHub520** [reply]
Hi, yes, self-trained custom YOLO11n models are supported. The model library only lists some common templates; we cannot and do not need to include every variant of every model there. You can follow the user_guide instructions to load your own model directly.

**Reply by klabklabgood** [reply]
Thank you for your reply!
